### ISSUE TITLE
Custom Image doesn't show up

### ISSUE THREAD

**Issue by nileshbohra** [RatelHub/rflutter_alert#108]
```
image: Image.asset(
                'assets/images/greentick.png',
              ),
```

**Comment by dryasbk** [comment]
Custom Image doesn't show up
even I tried the example

**Comment by shasank-periwal** [comment]
Yes, I tried that too
It didn't work
There's an issue with custom images.

**Comment by EduBarrio** [comment]
Yes, same error here. 

The error is on:

```
Widget _getImage() {
    Widget response = image ?? Container();
    switch (type) {
      case AlertType.success:
        response = Image.asset(
          '$kImagePath/icon_success.png',
          package: 'rflutter_alert',
        );
        break;
      case AlertType.error:
        response = Image.asset(
          '$kImagePath/icon_error.png',
          package: 'rflutter_alert',
        );
        break;
      case AlertType.info:
        response = Image.asset(
          '$kImagePath/icon_info.png',
          package: 'rflutter_alert',
        );
        break;
      case AlertType.warning:
        response = Image.asset(
          '$kImagePath/icon_warning.png',
          package: 'rflutter_alert',
        );
        break;
      case AlertType.none:
      default:
        response = Container();
        break;
    }
    return response;
  }
```

as you can see the response is a new Container when AlertType is no defined.

**Comment by JErazo7** [comment]
Yes, i confirm this issue

**Comment by nehal076** [comment]
Facing the same issue

**Comment by chaochaox1990** [comment]
add +1

**Comment by ibrahimdevs** [comment]
Sorry for this basic mistake, it is fixed on V2.0.3. Thank you all for pull requests.

**Comment by ahmad-dev7** [comment]
**rflutter_alert: ^2.0.7**
Custom image is not showing
